docs/docs/writing-documentation-with-docz.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Writing documentation with Docz"
+---
+
+Writing good documentation is important for your project maintainers (and for your future self!). A very nice documentation generator is [Docz](https://www.docz.site). It supports `mdx` files, which is short for Markdown with JSX. That means you can render React components in these special Markdown files. It can generate Prop tables and even provide a coding playground for your components!
+
+If you are starting your Gatsby project from scratch and would like to have Docz support out of the box, you can use the starter given in [Other Resources](#other-resources) below.
+
+Alternatively, the following guide should help you to get Docz working within an existing Gatsby project.
+
+## Setting up your environment
```

Comment:
Uhm, don't you miss the step of creating the Gatsby project àla `gatsby new my-awesome-gatsby-project` ? The docs should be approachable by everyone and it can be the case that some people see this doc page as their first doc page.

Reply:
> The docs should be approachable by everyone 

Absolutely, I overlooked this as this was a more specific use case. I will add the step of creating a Gatsby project a bit later. And noted, I will not use `we` when writing documentation, sorry about that! Thank you for your review!

Reply:
@LekoArts it should be fixed now. Let me know if you would like further changes.